### DISCUSSION TITLE
Creating new Flows have no Start shape

### DISCUSSION THREAD

**Discussion by mattnield** [postmanlabs/postman-flows#310] [Defect]
Using macOS (12.3.1) and the desktop client (9.17.1), and _new_ Flows that I create do not have a **Start** block and there does not seem to be an option to create one. I'm just left with a blank canvas and not running flows.

**Comment by HeathNaylor** [comment]
I also noticed there is no option to add a start block, but according to these release notes, they are removing the requirement on the start block.

https://github.com/postmanlabs/postman-flows/releases/tag/v22.05.1

**Reply by mattnield** [reply]
I did read that, but also noted that it takes about _'The hard requirement'_, which implies it should still be a thing to me 🙂

Possible its a bug that some blocks that _don't_ require input still have it as a mandatory thing.

**Comment by productman-josh** [comment]
If anyone has this issue, just start your flow with a 'String' block set to any value, then push the on-screen 'Start' button and it will begin your Flow at the 'String' block

**Reply by mattnield** [reply]
Thank you @productman-josh , that works.

I have to say it's an unintuitive approach, it would be better to put the Start block back in I think for when it's needed. In the cases where you can start Flow with a request, it's okay. But when - as in my case - you want to start a flow with some new Durables, it's really unclear what you need to do.

**Reply by hananwinner** [reply]
I also wanted to start with setting durables, and would prefer having explicit 'start' block.

I could now add a string block that became the starting point, but this is very un-intuitive.
